src/Microsoft.Health.Fhir.Shared.Core.UnitTests/Features/Search/Converters/FhirElementToSearchValueTypeConverterTests.cs

```diff
@@ -1,55 +0,0 @@
-﻿// -------------------------------------------------------------------------------------------------
-// Copyright (c) Microsoft Corporation. All rights reserved.
-// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
-// -------------------------------------------------------------------------------------------------
-
-using System;
-using System.Collections.Generic;
-using System.Linq;
-using Hl7.Fhir.Model;
-using Microsoft.Health.Fhir.Core.Features.Search.Converters;
-using Microsoft.Health.Fhir.Core.Features.Search.SearchValues;
-using Xunit;
-
-namespace Microsoft.Health.Fhir.Core.UnitTests.Features.Search.Converters
-{
-    public abstract class FhirElementToSearchValueTypeConverterTests<TTypeConverter, TElement>
-        where TTypeConverter : IFhirElementToSearchValueTypeConverter, new()
```

Comment:
>IFhirElementToSearchValueTypeConverter [](start = 31, length = 38)

I can be wrong, but I don't think we have unit tests for `IFhirNodeToSearchValueTypeConverter`
Only one I can see is `ResourceReferenceNodeToReferenceSearchValueTypeConverter`

I would advice to keep tests, but move them to `IFhirNodeToSearchValueTypeConverter`

There is still E2E tests, but I'm not sure we cover all possible scenarios.

To rephase: It's ok to delete SearchIndexTests, everything else I think need to be just repointed to new converter type, instead of removing them.